server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -690,7 +690,11 @@ linkageSection
 // -- local storage section ----------------------------------
 
 localStorageSection
-   : LOCAL_STORAGE SECTION DOT_FS dataDescriptionEntry*
+   : LOCAL_STORAGE SECTION DOT_FS dataDescriptionEntries
```

Comment:
It is correct, but why are you did it?

Reply:
It is just to have a special rule that parses only a set of variables, it's handy to parse copybooks.